tests/test_security_curve.cpp

```diff
@@ -40,6 +40,10 @@
 #  include <unistd.h>
 #endif
 
+#include "../src/tweetnacl.h"
```

Comment:
Sorry but I have some doubts about letting internal state&code leak in the tests, see README: https://github.com/zeromq/libzmq/blob/master/tests/README.md